ProvinceMapper/Source/Frames/Links/LinksTab.cpp

```diff
@@ -158,10 +160,19 @@ void LinksTab::leftUp(const wxGridEvent& event)
 		eventListener->QueueEvent(evt->Clone());
 
 		lastClickedRow = row;
+
+		// if the selected row's link is a comment
+		// highlight provinces from all links between the selected comment and the first comment below it
+		if (const auto& links = version->getLinks(); links->at(row)->getComment())
+		{
+			auto* evt = new wxCommandEvent(wxEVT_HIGHLIGHT_REGION);
+			evt->SetInt(row);
+			eventListener->QueueEvent(evt->Clone());
+		}
```

Comment:
For province map centering we click the link twice howevere here it would go into rename the comment. Maybe it's sufficient to just click once to both highlight and center the map on a first link inside the comment block?
However that requires the first link in block to be marked activeblock so I'm not sure how easy or relevant this is. Feel free to ignore if too much trouble.

Reply:
By activeblock you mean `activeRow` or `activeLink`?

Reply:
activeLink, since that one is sent to imagers to center on first pixel of first province within the link. This means you'd have to activate it even though comment itself is technically activated, which is why I said it's complicated. Either that or add code so that comment, already activated, is sent to imagers and when they realize it's a comment they center automatically on first row under it, assuming that's not a comment as well.